nbviewer/providers/base.py

```diff
@@ -658,30 +669,10 @@ def finish_notebook(self, json_notebook, download_url, msg=None,
             Notebook document in JSON format
         download_url: str
             URL to download the notebook document
-        provider_url: str, optional
-            URL to the notebook document upstream at the provider (e.g., GitHub)
-        provider_icon: str, optional
-            CSS classname to apply to the navbar icon linking to the provider
-        provider_label: str, optional
-            Text to to apply to the navbar icon linking to the provider
         msg: str, optional
             Extra information to log when rendering fails
-        breadcrumbs: list of dict, optional
-            Breadcrumb 'name' and 'url' to render as links at the top of the notebook page
         public: bool, optional
             True if the notebook is public and its access indexed, False if not
-        format: str, optional
-            Rendering format (e.g., script, slides, html)
-        request: tornado.httputil.HTTPServerRequest, optional
-            HTTP request that triggered notebook rendering
-        title: str, optional
-            Title to use as the HTML page title (i.e., text on the browser tab)
-        executor_url: str, optional
-            URL to execute the notebook document (e.g., Binder)
-        executor_label: str, optional
-            Text to apply to the navbar icon linking to the execution service
-        executor_icon: str, optional
```

Comment:
Are all of these parameters now in `namespace`? If so, we need to document them somewhere for reference. Leaving the parameter docstrings here but noting they are keys in `namespace` instead of true parameters is one approach. Feel free to suggest another one.